waku/v2/protocol/filter/client.go

```diff
@@ -160,15 +161,17 @@ func (wf *WakuFilterLightNode) onRequest(ctx context.Context) func(s network.Str
 			return
 		}
 
-		if !wf.subscriptions.Has(s.Conn().RemotePeer(), messagePush.PubsubTopic, messagePush.WakuMessage.ContentTopic) {
-			logger.Warn("received messagepush with invalid subscription parameters", logging.HostID("peerID", s.Conn().RemotePeer()), zap.String("topic", messagePush.PubsubTopic), zap.String("contentTopic", messagePush.WakuMessage.ContentTopic))
+		if !wf.subscriptions.Has(s.Conn().RemotePeer(), *messagePush.PubsubTopic, messagePush.WakuMessage.ContentTopic) {
+			logger.Warn("received messagepush with invalid subscription parameters",
+				logging.HostID("peerID", s.Conn().RemotePeer()), zap.String("topic", *messagePush.PubsubTopic),
```

Comment:
`messagePush.PubsubTopic` can be nil, so this dereference could fail. 
Perhaps we should add something like this in logging/logging.go:
```go

func NullString(key string, value *string) zap.Field {
	v := "<nil>"
	if value != nil {
		v = *value
	}
	return zap.String(key, v)
}
```
and then use instead  `logging.NullString("topic", messagePush.PubsubTopic)`